charts/refarch-templates/Chart.yaml

```diff
@@ -2,13 +2,13 @@ apiVersion: v2
 name: refarch-templates
 description: Helm Chart for deploying a it@M Reference Architecture application.
 type: application
-version: 1.0.14
+version: 1.0.15
 home: https://github.com/it-at-m/helm-charts/tree/main/charts/refarch-templates
-icon: https://assets.muenchen.de/logos/itm/itm-logo-256.png
+icon: https://assets.muenchen.de/logos/itm/itM_Basislogo_gelb_schwarz-256.png
 dependencies:
   - name: refarch-gateway
     condition: refarch-gateway.enable
-    version: 1.6.1
+    version: 1.6.2
```

Comment:
@freinold you can't bump this version as 1.6.2 is not yet published (thus CI fails). after this PR is merged and the charts are released, renovate should pick up this version bump in a following PR